superset/security/manager.py

```diff
@@ -994,8 +994,14 @@ def raise_for_access(  # pylint: disable=too-many-arguments,too-many-branches
 
             assert datasource
 
+            from superset import is_feature_enabled
```

Comment:
I think this is the old way of importing. I believe the correct way is
```suggestion
            from superset.extensions import feature_flag_manager
```
and then doing `feature_flag_manager.is_feature_enabled`